rero_ils/modules/vendors/api.py

```diff
@@ -76,9 +76,8 @@ def order_email(self):
         vendor = self.replace_refs()
         if vendor:
             return vendor.get(
-                'order_contact', vendor.get(
-                    'default_contact', {})).get('email')
-        return None
+                'order_contact', vendor.get('default_contact', {})
+            ).get('email')
```

Comment:
what a strange construction... the `replace_refs` seems completely unnecessary.
```python
@property
def order_email(self):
    """Shortcut for vendor order email."""
    return self.get('order_contact', self.get('default_contact', {})).get('email')
```

Reply:
Yes indeed, very strange.